services/datamanager/builtin/builtin.go

```diff
@@ -140,6 +140,8 @@ type builtIn struct {
 
 	syncSensor           selectiveSyncer
 	selectiveSyncEnabled bool
+
+	statusChangedMap map[string]interface{}
```

Comment:
It's a good idea to keep a map in order to only log when something change, similar to the `syncDisabled` and `syncIntervalMins` variables above which track this at the service level. To make this descriptive, how about something like `componentMethodIsDisabled`, and can switch to a map[string]bool?

Reply:
Doing this as a bool will only allow us to keep track of on/off. If we do a map of string -> interface then it allows us to track any changes (and log whenever something changes)

Reply:
Ah yep, perhaps still something descriptive like `componentMethodFrequencyHz` and have the values be of type float32 rather than interface, so that it's clear what this map represents.